apps/sms/style/sms.css

```diff
@@ -443,7 +443,7 @@ section[role="region"].new > header:first-child form {
 */
 
 .carrier-wrapper {
-  display: none;
+  display: block;
```

Comment:
I think you need to keep this, because there are cases where we want to hide it and we wantto display it only in some cases